app/routes/_index.tsx

```diff
@@ -1,22 +1,90 @@
-import { Card, CardGroup, CardHeader } from "@trussworks/react-uswds";
+import {
+    Breadcrumb,
+    BreadcrumbBar,
+    BreadcrumbLink,
+    Card,
+    CardGroup,
+    CardHeader,
+    Header,
+    Label,
+    Link,
+    Select,
+} from "@trussworks/react-uswds";
+import React from "react";
 import { mergeMeta } from "~/lib/merge-meta";
 
 export const meta = mergeMeta(({ parentTitle }) => [{ title: `Meeting Spaces • ${parentTitle}` }]);
 
 export default function Index() {
     return (
-        <CardGroup>
-            <Card>
-                <CardHeader>
-                    <h1 className="usa-card__heading">Meeting Spaces</h1>
-                    <p className="font-sans text-base-darker">
-                        Austin Public Library Meeting Spaces are
-                        <strong className="font-bold"> free of charge</strong> and ideal for
-                        discussion groups, panels, and lectures. Both paper and online reservation
-                        requests are timestamped and processed in the order they are received.
-                    </p>
-                </CardHeader>
-            </Card>
-        </CardGroup>
+        <>
+            <div className="flex justify-center">
+                <div className="min-w-[30rem] max-w-[62rem]">
```

Comment:
Same thing. This only contains the child CardGroup. We can remove this and apply the styling directly to CardGroup